arviz/plots/bpvplot.py

```diff
@@ -134,14 +135,12 @@ def plot_bpv(
         Passed to :meth:`matplotlib.axes.Axes.plot` or
         :meth:`matplotlib.axes.Axes.axhspan` (when ``kind=u_value``
         and ``reference=analytical``).
-    backend_kwargs : bool, optional
+    backend_kwargs : bool, optional 
```

Comment:
You will need to review the trailing whitespace, otherwise the auto code formatting check won't pass.

Reply:
if I got it right is it to leave a space at the end?